ogr/services/forgejo/comments.py

```diff
@@ -0,0 +1,193 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+import datetime
+from typing import Any
+
+from ogr.abstract import Comment, CommitComment, IssueComment, PRComment, Reaction
+
+
+class ForgejoReaction(Reaction):
+    _raw_reaction: dict
+
+    def __init__(
+        self,
+        raw_reaction: dict,
+        forgejo_client: Any,
+        repository: str,
+        issue_number: int,
+    ) -> None:
```

Comment:
Where does this `__init__` come from. I don't see a reference to `forgejo_client` in the current codebase